msal-java-articles/advanced/managed-identity.md

```diff
@@ -31,7 +31,7 @@ For a complete list, refer to [Azure services that can use managed identities to
 
 MSAL libraries provide lower level APIs that are closer to the OAuth2 and OIDC protocols. 
 
-Both MSAL Java and [Azure SDK](/dotnet/api/overview/azure/identity-readme?view=azure-dotnet&preserve-view=true) allow to acquire tokens via managed identity. Internally, Azure SDK uses MSAL Java, and it provides a higher-level API via its `DefaultAzureCredential` and `ManagedIdentityCredential` abstractions. 
+Both MSAL Java and [Azure SDK](/azure/developer/java/sdk/overview) allow to acquire tokens via managed identity. Internally, Azure SDK uses MSAL Java, and it provides a higher-level API via its `DefaultAzureCredential` and `ManagedIdentityCredential` abstractions. 
```

Comment:
nit: "...allow to acquire tokens..." should probably be "...allow **you** to acquire tokens..."

Reply:
Or, alternatively, "acquiring tokens"

Reply:
```suggestion
Both MSAL Java and [Azure SDK](/azure/developer/java/sdk/overview) allow acquiring tokens via Managed Identity. Internally, Azure SDK uses MSAL Java. It provides a higher-level API via its <xref:com.azure.identity.DefaultAzureCredential> and <xref:com.azure.identity.ManagedIdentityCredential> abstractions. 
```